backend/src/migrations/20230827100608_remove_password_salt.ts

```diff
@@ -0,0 +1,17 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'users';
+
+function up(knex: Knex): Promise<void> {
+    return knex.schema.table(TABLE_NAME, function (table) {
+        table.dropColumn('password_salt');
```

Comment:
Create 
const ColumnName = {
    PASSWORD_SALT: 'password_salt',
};

and use it instead, because you use it in two different places

Reply:
Done